base/applications/rapps/appview.cpp

```diff
@@ -1279,6 +1303,12 @@ HWND CAppsListView::Create(HWND hwndParent)
     SetImageList(m_hImageListView, LVSIL_SMALL);
     SetImageList(m_hImageListView, LVSIL_NORMAL);
 
+#pragma push_macro("SubclassWindow")
+#undef SubclassWindow
+    m_hWnd = NULL;
+    SubclassWindow(hwnd);
+#pragma pop_macro("SubclassWindow")
```

Comment:
What's happening there? Is it colliding with ATL definition?

Reply:
`#define SubclassWindow(hwnd,lpfn) ((WNDPROC)SetWindowLongPtr((hwnd),GWLP_WNDPROC,(LPARAM)(WNDPROC)(lpfn)))`